assets/style.css

```diff
@@ -0,0 +1,998 @@
+
+.sections {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-size: 18px;
+  line-height: 18px;
+  color: black;
+}
+
+.sections > li {
+  flex-grow: 1;
+  border-left: 1px solid #d9d9d9;
+  text-align: center;
+  margin: auto;
+}
+
+.nav-bottom {
+  border-bottom: solid 2px #f4f5f6;
+  height: 56px;
+}
+
+.search {
+  color: rgba(3, 3, 3, 0.4);
+  font-size: 16px;
+  font-family: Arial, sans-serif;
+  font-weight: 500;
+  width: 70px;
+}
+
+.search-box {
+  width: 70px;
+  border: none;
+}
+
+.search-wrap {
+  width: 168px;
+  height: 35px;
+}
+
+li.search-box-wrapper {
+  flex-grow: 0 !important;
+}
+
+.search-box:focus {
+  position: absolute;
+  left: 10px;
+  width: 100%;
+  top: -2px;
+}
+
+.search-icon {
+  background: url(images/search-icon.svg) no-repeat;
+  width: 105px;
+  height: 20px;
+  margin-top: 20px;
+  border: none;
+  margin-left: 12px;
+}
+
+.editions > li,
+.footer-bottom > li {
+  border: 0 10px;
+  padding-left: 10px;
+  padding-right: 10px;
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+  height: 15px;
+  margin-top: 10px;
+}
+
+.search-wrap > .search-icon,
+.search-wrap > .search,
+.sections > li:nth-child(1) {
+  border: none;
+}
+
+.nav-bottom::before {
+  content: "";
+  display: block;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 80px;
+}
+
+/* top section */
+
+.story-section {
+  color: black;
+  font-size: 16px;
+  font-weight: bold;
+}
+
+.story-tag {
+  background-color: rgba(255, 255, 255, 0.9);
+  color: #f72210;
+  font-weight: bold;
+  line-height: 16px;
+  padding: 8px 15px 9px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.main-articles figcaption {
+  font-size: 22px;
+  line-height: 24px;
+  font-weight: bold;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  color: black;
+}
+
+.story-description {
+  color: #333;
+  font-size: 13px;
+  line-height: 18px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: initial;
+  padding-top: 15px;
+}
+
+.vs-circle {
+  border-radius: 50%;
+  font-size: 14px;
+  font-weight: 700;
+  z-index: 1;
+  height: 32px;
+  width: 32px;
+  background-color: black;
+  color: #d6d6d6;
+  text-align: center;
+  line-height: 32px;
+}
+
+.vs-decoration::before {
+  display: block;
+  content: "";
+  height: 15px;
+  width: 100%;
+  position: absolute;
+  border-bottom: solid 1px #d6d6d6;
+}
+
+.vs-decoration::after {
+  display: block;
+  content: "";
+  height: 17px;
+  width: 100%;
+  position: absolute;
+  border-bottom: solid 1px #d6d6d6;
+}
+
+.circ-pic {
+  width: 63px;
+  height: 63px;
+  flex-basis: 63px;
+}
+
+.debate .title,
+.opinion .title {
+  font-size: 17px;
+  line-height: 21px;
+  color: black;
+}
+
+.debate-first {
+  border-top: #f92859 4px solid;
+}
+
+.debate-second {
+  border-bottom: #1c87fb 4px solid;
+}
+
+.by-line {
+  font-size: 14px;
+  line-height: 21px;
+  text-transform: uppercase;
```

Comment:
we can use the `text-uppercase` bootstrap class here

Reply:
Done! ; )